wasm/__init__.py

```diff
@@ -4602,45 +4599,62 @@ def type_table(store, tableaddr):
 
 def read_table(store, tableaddr, i):
     if len(store["tables"]) < tableaddr:
-        return "error"
+        raise ValidationError(
```

Comment:
Huh, the other tests like `type_table` were testing `<=`. Not sure which one is right, but they should almost certainly be the same.

I guess that will get taken care of in the the follow-up refactor you mentioned. But maybe a `#TODO` note until then.